.eslintrc.json

```diff
@@ -26,6 +26,12 @@
       "extends": ["plugin:@nrwl/nx/typescript"],
       "rules": { "@typescript-eslint/ban-ts-comment": "off" }
     },
+    {
+      "files": ["*.spec.ts", "*.spec.tsx", "*.spec.js", "*.spec.jsx"],
+      "rules": {
+        "@nrwl/nx/enforce-module-boundaries": "off"
```

Comment:
I wonder, is this really what we want? Another solution would be to extract the Lumberjack interfaces/domain into a separate library and make the internal libraries import from there.

That would really remove the circular dependencies.

Reply:
More namespaces for core concerns would decrease the developer experience for users of the library in my opinion.